src/backup/BackupComplete.tsx

```diff
@@ -32,7 +32,6 @@ function BackupComplete({ route }: Props) {
         // TODO(ACT-1133): change settingsScreen props to isAccountRemoval boolean to know if we need to go back to settings to show promptConfirmRemovalModal
         navigate(settingsScreen === Screens.Settings ? Screens.Settings : Screens.SettingsDrawer, {
```

Comment:
```suggestion
        navigate(settingsScreen, {
```

Reply:
doing this I get the following error:
```
Argument of type '[Screens, { promptConfirmRemovalModal: true; }]' is not assignable to parameter of type '[keyof StackParamList] | [keyof StackParamList, {} | { settingsScreen: Screens | undefined; } | { nextScreen: keyof StackParamList; } | ... 68 more ... | undefined]'.
  Type '[Screens, { promptConfirmRemovalModal: true; }]' is not assignable to type '[keyof StackParamList, {} | { settingsScreen: Screens | undefined; } | { nextScreen: keyof StackParamList; } | { activeAssetTab: AssetTabType; isWalletTab?: boolean | undefined; } | ... 67 more ... | undefined]'.
    Type at position 0 in source is not compatible with type at position 0 in target.
      Type 'Screens' is not assignable to type 'keyof StackParamList'.
```

Reply:
Ah, I think you can avoid this if you narrow the type of settingsScreen to `Screens.Settings | Screens.SettingsDrawer`